src/nvidia-cuda/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+install_cuda=${INSTALL_CUDA:-"true"}
+install_cudnn=${INSTALL_CUDNN:-"true"}
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    exit 1
+fi
+
```

Comment:
You are missing logic to check for the commands you are using and then installing them if not present.  It looks like you'd need `wget` and `apt-transport-https` at a minimum which may not be in base images. You may also need `gnupg2`. 

Try using this feature with the straight `ubuntu` and `debian` images to verify what you can count on being present.

Codespces' kitchen sink is massive partly because all of this is already there.

Reply:
Thanks! I added the `apt_get_update_if_needed` and `check_packages` functions from the Python feature. Adding `check_packages wget ca-certificates` fixes things on `ubuntu` and `debian`.